lib/BackendClient.js

```diff
@@ -73,7 +73,7 @@ var BackendClient = (function(){
                     client.setSessionId(results);
                     deferred.resolve();
                 }
-            }, adminSecret, null, kalturaTypes.KalturaSessionType.ADMIN, partnerId, null, null);
+            }, adminSecret, null, kalturaTypes.KalturaSessionType.ADMIN, partnerId, null, "all:*,disableentitlement");
```

Comment:
i think it's better to add this  into the config.template 
"backendClient" : {
    "adminSecret" :"admin secret",
    "partnerId" : 12345,
    "serviceUrl" : "http://serviceURL",
    "ksSessionRefreshIntervalMinutes" : 720,
    "ksPrivileges":  "all:*,disableentitlement"
  },